### ISSUE TITLE
[Catalog] Snackbar demo cells cannot be accessed by Switch Control

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4712]
The cells that allow for activation to pop-up a Snackbar message are inaccessible to Switch Control. It only navigates the AppBar elements.

## Reproduction steps
1. Turn on Switch control
1. Open the catalog &rarr; Snackbar &rarr; Start Demo
1. Navigate through the on-screen elements

## Expected behavior
The table of Snackbar message cells is accessible.

## Actual behavior
The table of Snackbar message cells is inaccessible for Switch Control.

## Platform (please complete the following information)
iPhone 5c/iOS 10.3.
